build.gradle

```diff
@@ -15,7 +15,7 @@ plugins {
 }
 
 ext {
-    springBootVersion = "${System.getenv("SPRING_BOOT_VERSION") ?: '3.2.2'}"
+    springBootVersion = "${System.getenv("SPRING_BOOT_VERSION") ?: '3.4.0'}"
```

Comment:
Should it be,
```suggestion
    springBootVersion = "${System.getenv("SPRING_BOOT_VERSION") ?: '3.3.1'}"
```

Reply:
Good catch. I had this in for catching local build issues but forgot to revert it back